### ISSUE TITLE
Having your work recognized in our AboutWindow

### ISSUE THREAD

**Issue by wwmm** [wwmm/easyeffects#1949]
I am opening this issue to make it easier for contributors to manifest their desire of having their names cited in our about window page. Although I could just take a look at Github's `Insight` tab not everybody wants to have their names exposed to the masses.  And maybe they do want but in a different way. And on top of these factors I may just forget about adding the name in the AboutWindow because I usually do not pay much attention to it.

So if you have ever contributed to this project and would like to have your name shown in our AboutWindow feel free to reply in this issue :-).

**Comment by Digitalone1** [comment]
@LebedevRI made the porting of the LSP Gate. Maybe they are interested.

**Comment by LebedevRI** [comment]
I don't believe that was a significant-enough change. (LOC is not a relevant metric)
